mantidproject/mantidimaging
# Enable drag and drop loading for

### Issue
Closes #778 

### Description
Allows a user to drag a folder containing a sample onto the program and it will load in the stacks.

### Testing
- Open imaging
- Drag on a folder containing a sample + flat (before/after) + dark (before/after) + monitor normalisation
- The sample should be loaded fine
- Try to break the sample, by running a full recon, performing monitor normalization and all standard operations pre-recon.

### Documentation
No
